ci-operator/step-registry/openshift/microshift/infra/rpm-install-aws/openshift-microshift-infra-rpm-install-aws-commands.sh

```diff
@@ -17,7 +17,11 @@ EOF
 
 configure_vm_args=""
 if "${OPTIONAL_RPMS}"; then
-  configure_vm_args="--optional-rpms"
+  configure_vm_args="--skip-optional-rpms"
```

Comment:
I'm not sure we want to change this block. We want to install optional RPMs if the setting is true, right?